core/store/orm/config.go

```diff
@@ -174,6 +174,14 @@ func (c Config) getDuration(s string) models.Duration {
 	return rv
 }
 
+func (c Config) DatabaseListenerMinReconnectInterval() time.Duration {
+	return c.viper.GetDuration(EnvVarName("DatabaseListenerMinReconnectInterval"))
```

Comment:
It seems like we don't actually use this? It looks hardcoded to me

Reply:
Nah, check `core/services/chainlink/application.go` line 106:
```go
eventBroadcaster := postgres.NewEventBroadcaster(config.DatabaseURL(), config.DatabaseListenerMinReconnectInterval(), config.DatabaseListenerMaxReconnectDuration())
```